src/garage/torch/algos/_utils.py

```diff
@@ -158,4 +158,4 @@ def filter_valids(tensor, valids):
         torch.Tensor: Filtered Tensor
 
     """
-    return [tensor[i][:valids[i]] for i in range(len(valids))]
+    return [tensor[i][:valid] for i, valid in enumerate(valids)]
```

Comment:
if there was a bug here, please update the  test to catch it

Reply:
No, I just want to refactor the code